### PR TITLE
pkcs1: export RSAPSS_SALT_LEN_DEFAULT

### PR DESCRIPTION
**Pull request by lumag**
Export default RSA PSS salt length.

### CODE REVIEW

File: pkcs1/src/params.rs [hunk 1]
```diff
@@ -17,7 +17,8 @@ const SHA_1_AI: AlgorithmIdentifierRef<'_> = AlgorithmIdentifierRef {
     parameters: None,
 };
 
-const SALT_LEN_DEFAULT: u8 = 20;
+/// Default RSA PSS Salt length in RsaPssParams
+pub const RSAPSS_SALT_LEN_DEFAULT: u8 = 20;
```
Comment (tarcieri) [inline pkcs1/src/params.rs:21]: Maybe use an inherent constant like `RsaPssParams::SALT_LEN_DEFAULT`?

**Comment by lumag** [comment]
ack, sounds good

File: pkcs1/src/params.rs (hunk 1 above)
Comment (lumag) [reply pkcs1/src/params.rs]: Done

**Comment by tarcieri** [comment]
Thanks!

**Pull request merged**